drivers/adc/ad9250/ad9250.c

```diff
@@ -616,28 +561,18 @@ int32_t ad9250_output_invert(struct ad9250_dev *dev,
  *		     Example: 0 � offset binary(default);
  *			      1 � two's complement;
  *
- * @return Returns negative error code or the set output format.
+ * @return Returns negative error code or 0 in case of success.
  *******************************************************************************/
 int32_t ad9250_output_format(struct ad9250_dev *dev,
 			     int32_t format)
 {
-	uint32_t ret = 0;
-
-	if( (format == 0) || (format == 1)) {
-		ret = ad9250_set_bits_to_reg(dev,
-					     AD9250_REG_OUT_MODE,
-					     AD9250_OUT_MODE_DATA_FORMAT(format),
-					     AD9250_OUT_MODE_DATA_FORMAT(1));
-	} else {
-		ret = ad9250_read(dev,
-				  AD9250_REG_OUT_MODE);
-		if(ret != SUCCESS) {
-			return ret;
-		}
-		return (ret & AD9250_OUT_MODE_DATA_FORMAT(1)) >> 0;
-	}
-
-	return ret;
+	if( (format == 0) || (format == 1))
+		return ad9250_set_bits_to_reg(dev,
+					      AD9250_REG_OUT_MODE,
+					      AD9250_OUT_MODE_DATA_FORMAT(format),
+					      AD9250_OUT_MODE_DATA_FORMAT(1));
+	else
```

Comment:
else clause is not needed you can just return FAILURE

Reply:
or a better approach will be:
```suggestion
if(!(format == 0 || format == 1))
     return FAILURE;
     
return ad9250_set_bits_to_reg(...)

```